benefits/eligibility/analytics.py

```diff
@@ -4,37 +4,45 @@
 from benefits.core import analytics as core
 
 
-class StartedEligibilityEvent(core.Event):
+class EligibilityEvent(core.Event):
+    """Base analytics event for eligibility verification."""
+
+    def __init__(self, request, event_type, eligibility_types):
+        super().__init__(request, event_type)
+        self.update_event_properties(eligibility_types=eligibility_types)
+
+
+class StartedEligibilityEvent(EligibilityEvent):
     """Analytics event representing the beginning of an eligibility verification check."""
 
-    def __init__(self, request):
-        super().__init__(request, "started eligibility")
+    def __init__(self, request, eligibility_types):
+        super().__init__(request, "started eligibility", eligibility_types)
 
 
-class ReturnedEligibilityEvent(core.Event):
+class ReturnedEligibilityEvent(EligibilityEvent):
     """Analytics event representing the end of an eligibility verification check."""
 
-    def __init__(self, request, status, error=None):
-        super().__init__(request, "returned eligibility")
+    def __init__(self, request, eligibility_types, status, error=None):
+        super().__init__(request, "returned eligibility", eligibility_types)
         if str(status).lower() in ("error", "fail", "success"):
             self.update_event_properties(status=status, error=error)
 
 
```

Comment:
All the of the helper functions now require an additional parameter for the types